.github/workflows/linting.yml

```diff
@@ -0,0 +1,17 @@
+name: Linting
+
+on:
+  push:
+    branches:
+      - main
+      - "renovate/**"
+  pull_request:
+
+jobs:
+  linting:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: UCL-MIRSG/.github/actions/linting@v0.25.0
+        with:
+          ansible-roles-config: ./requirements.yml
```

Comment:
Do we in general have a `requirements.yml` file? I don't know, but I found one here, for example: https://github.com/UCL-MIRSG/ansible-role-postgresql/blob/main/meta/requirements.yml

Reply:
I think the requirements file would generally be in `meta/requirements.yml`, but not all roles have dependencies. This template hasn't got a requirements file, so perhaps remove from the linting config?

Reply:
So here I've added one https://github.com/UCL-MIRSG/mirsg-template/blob/main/requirements.yml specifically, so it's obvious for the `linting.yml` file. When not all repos would have `ansible`.

In this case, I'd be tempted to move it into `meta/requirements.yml`. One can just delete if not relevant.